lib/rubocop/cop/rails/env.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks for usage of `Rails.env` which can be replaced with Feature Flags
+      #
+      # @example
+      #
+      #   # bad
+      #   Rails.env.development? || Rails.env.production? || Rails.env.local?
```

Comment:
`local` already encompasses `development`. Can you write this differently?

Reply:
How does `Rails.env.development? || Rails.env.production? || Rails.env.test?` look? Or did you have something particular in mind?

Reply:
Sure, that seems fine. You could also drop `development` instead, I don't have a preference.